internal/nostr/nostr.go

```diff
@@ -655,13 +657,13 @@ func (svc *Service) StopSubscriptionHandler(c echo.Context) error {
 	}
 
 	svc.Logger.WithFields(logrus.Fields{
-		"subscription_id": subscription.ID,
+		"subscription_id": subscription.Uuid,
 	}).Debug("Stopping subscription")
 
 	err := svc.stopSubscription(&subscription)
 	if err != nil {
 		svc.Logger.WithFields(logrus.Fields{
-			"subscription_id": subscription.ID,
+			"subscription_id": subscription.Uuid,
```

Comment:
can we log an error rather than debug when something unexpected like this happens?